ui/oopsyraidsy/data/06-ew/dungeon/the_tower_of_babil.ts

```diff
@@ -0,0 +1,56 @@
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export type Data = OopsyData;
+
+// TODO: Figure out how to determine whether someone actively stood in Boundless Pain
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.TheTowerOfBabil,
+  damageWarn: {
+    'Babil Nimrod Cannon Shot': '6DE1', // Large circle AoE, before first boss
+    'Babil Tempered Imperial Spread Shot': '6DE2', // Conal AoE, before first boss
+    'Babil Satellite Incendiary Circle': '6DE3', // Large donut AoE, before first boss
+    'Babil Reaper Magitek Cannon': '6DE4', // AOE circle, before first boss
+
+    'Babil Barnabas Electromagnetic Release 1': '62F1', // Center AoE circle, boss 1
+    'Babil Barnabas Electromagnetic Release 2': '62EF', // Center magnetic line, boss 1
+    'Babil Barnabas Rolling Scrapline': '62EB', // Center AoE circle, no magnets, boss 1
+    'Babil Thunderball Shock': '62F2', // Cardinal/intercardinal AoE circles, boss 1
+
+    'Babil Rearguard Cermet Pile': '6DE8', // Line AoE, before boss 2
+    'Babil Colossus Grand Sword': '6DE9', // Frontal cone, before boss 2
+    'Babil Avenger Shoulder Cannon': '6DEA', // AoE circle, before boss 2
+
+    'Babil Magitek Chakram Mighty Blow': '62F4', // Arena AoE, if not mini, boss 2
+    'Babil Lugae Surface Missile': '62F7', // Targeted AoE circles from Magitek Missile, boss 2
+    'Babil Magitek Explosive Explosion': '62F9', // Bomberman line AoE, boss 2
+    'Babil Lugae Magitek Ray': '62FC', // Frontal line AoE, boss 2
+
+    'Babil Zot Sky Armor Garlean Fire': '5E4A', // Circle AoE, after boss 2
+    'Babil Armored Weapon Diffractive Laser': '5E53', // Circle AoE, after boss 2
+    'Babil Magitek Crane Crane Game': '6C35', // Large environmental circle AoE, after boss 2
+    'Babil Hexadrone 2-Tonze Magitek Missile': '6DEDC', // Circle AoE, after boss 2
```

Comment:
I don't actually have any logs where this appears, but it was in a list of mechanics quisquous handed me a while ago. I guessed it was this ID since the numbers line up.

Reply:
Yeah, you're correct that it's `6DEC`.